README.md

```diff
@@ -59,49 +59,21 @@ poetry run python <script-path>
 
 ### Cardano Node and Ogmios
 
-Minimum Specs for Preview Network:
-- 2 Core CPU
-- 4GB memory
-- 16GB free storage
-
-First install Docker.
-Follow the official documentation [here](https://docs.docker.com/get-docker/).
-
-
-To start a [Cardano Node](https://github.com/input-output-hk/cardano-node) and [Ogmios API](https://ogmios.dev/) use [docker-compose](https://docs.docker.com/get-started/08_using_compose/) in your terminal:
-
-```bash
-# make sure your node configurations are up to date
-git submodule update --init
-# starts a cardano node and ogmios api on the preview testnet
-docker compose up
-```
-
-You can then access the `cardano-cli` using the docker image:
-```bash
-docker run --rm -it \
-  --entrypoint cardano-cli \
-  -e CARDANO_NODE_SOCKET_PATH=/ipc/node.socket \
-  -v opshin-pioneer-program_node-db:/data \
-  -v opshin-pioneer-program_node-ipc:/ipc \
-  inputoutput/cardano-node
-```
+#### Quick setup
```

Comment:
Could you add an explanation of chain synchronization and how third party hosted nodes work here?
Note that production apps should host their own nodes.

Reply:
Not sure what to put there. Any reference documentation? These nodes are usually already fully synced and just hosted by a third party, what else to add? :sweat_smile:

Reply:
added something :)